src/commands/force/user/permset/assign.ts

```diff
@@ -60,12 +60,12 @@ export class UserPermsetAssignCommand extends SfdxCommand {
         const queryResult = await connection.singleRecordQuery<{ Id: string }>(
           `SELECT Id FROM User WHERE Username='${username}'`
         );
-
+        const permsetArray = this.flags.permsetname as string[];
```

Comment:
I'm not sure if there's a standard around this, but it's been suggested to me to use `asArray` or `asArray<string>` if necessary instead of `as string[]`

Reply:
Happy to change it, this pattern is also used at line 50 and in other files in the code base. 
Want me to change others?

Reply:
eslint complains at me when I run build 
```
63:56  error  Array type using 'Array<string>' is forbidden for simple types. Use 'string[]' instead  @typescript-eslint/array-type
```
I guess I should just leave it?